presto-iceberg/src/main/java/com/facebook/presto/iceberg/rest/IcebergRestCatalogFactory.java

```diff
@@ -67,6 +68,11 @@ protected Map<String, String> getCatalogProperties(ConnectorSession session)
 
         catalogConfig.getAuthenticationType().ifPresent(type -> {
             if (type == OAUTH2) {
+                // The oauth2 endpoint of the REST catalog spec has been deprecated and will
```

Comment:
From the iceberg PR, they specify that it is not the entire oauth2 endpoint, but the `oauth2/tokens` endpoint which is deprecated. I think we should be a little more specific.

Reply:
yes, good point